src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.h

```diff
@@ -33,6 +33,8 @@ class MTROTAProviderDelegateBridge : public chip::app::Clusters::OTAProviderDele
     // touches Matter objects.
     void Shutdown();
 
+    void ResetState();
```

Comment:
Where is this implemented?  I don't see it....

Reply:
i moved it to the public section. If its not implemented i won't be able to compile ;)

Reply:
That's on a different class, not the bridge class.

Reply:
oh yes in the BDXOtaSender